es6-collections/es6-collections.d.ts

```diff
@@ -0,0 +1,113 @@
+// Type definitions for es6-collections v0.5.1
+// Project: https://github.com/WebReflection/es6-collections/
+// Definitions by: Ron Buckton <http://github.com/rbuckton>
+// Definitions: https://github.com/borisyankov/DefinitelyTyped
+
+/* *****************************************************************************
+Copyright (c) Microsoft Corporation. All rights reserved.
+Licensed under the Apache License, Version 2.0 (the "License"); you may not use

```

Comment:
This repository under the MIT license. https://github.com/borisyankov/DefinitelyTyped/blob/master/LICENSE
Where is this definition file from?